_posts/2024-07-22-the-x-files.md

```diff
@@ -0,0 +1,1141 @@
+---
+layout: post
+title: The X-Files - Or which project interface to pick?
```

Comment:
The title could reflect that this is comparing build/automation tools.
"project interface" makes me think of JIRA and other project management tooling